.github/workflows/npm-publish.yml

```
@@ -0,0 +1,74 @@
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      # Get yarn cache directory and set it in a variable
+      - name: Get yarn cache path
+        id: yarn-cache-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      
+      # Checkout branch or head commit
+      - name: Checkout branch
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)